openapi/external.yaml

```diff
@@ -0,0 +1,162 @@
+openapi: 3.0.1
+info:
+  version: 1.0.0
+  title: IO Lollipop Function Lollipop Consumer API
+  x-logo:
+    url: https://io.italia.it/assets/img/io-logo-blue.svg
+  description: >
+    Documentation of the IO Lollipop Function API exposed to Lollipop
+    Consumerhere.
+servers:
+  - url: https://api.pagopa.it/api/v1/lollipop
+security:
+  - ApiKeyAuth: []
+  - BearerAuth: []
+paths:
+  /assertions/{assertion_ref}:
+    get:
+      operationId: getAssertion
+      summary: Get Assertion related to a give assertion ref
+      parameters:
+        - name: assertion_ref
+          required: true
+          in: path
+          schema:
+            $ref: '#/components/schemas/AssertionRef'
+      responses:
+        '200':
+          description: The assertion related to a valid assertion_ref
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LCAssertion'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+        '409':
+          description: Assertion is not valid yet
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+components:
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: Ocp-Apim-Subscription-Key
+    BearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
```

Comment:
This type for securitySchema will be applied to `Authentication` header? Should we use a custom header name?